src/components/dex/MarketMegaSelector.vue

```diff
@@ -67,7 +69,7 @@ export default {
         let nonZeroVolumeItems = 0;
         markets.forEach((market) => {
           const baseAsset = this.$services.neo.getHolding(market.baseAssetId);
-          const unitValue = baseAsset ? baseAsset.unitValue : 0;
+          const unitValue = baseAsset ? baseAsset.unitValue : market.unitValue || 0;
```

Comment:
This unit value needs to be the market's base asset unit value, it seems you have set `market.unitValue` to the quote asset unit value.

Reply:
It would also be better to name it `market.baseUnitValue`